Sources/Xcodeproj/pbxproj.swift

```diff
@@ -405,7 +407,7 @@ public func xcodeProject(
         // Create a Xcode target for the target.
         let package = packagesByTarget[target]!
         let xcodeTarget = project.addTarget(
-            objectID: "\(package.manifestName)::\(target.name)", // TODO: use identity instead?
+            objectID: "\(package.manifest.displayName)::\(target.name)", // TODO: use identity instead?
```

Comment:
@neonichu @abertelrud I would have liked to use identity here but not sure about the downstream effects. wdyt?

Reply:
All object IDs should use the package ID, I think.